release-notes/6.0/linux-packages.md

```diff
@@ -0,0 +1,81 @@
+# Linux package dependencies
+
+.NET 6 has several dependencies that must be satisfied to run .NET apps. The commands to install these libraries are listed for multiple distributions.
+
+Feel free to contribute packages for distributions not (yet) listed in this document, including ones not supported by the .NET Team.
+
+Tips:
+
+- [runtime-deps container images](https://github.com/dotnet/dotnet-docker/tree/main/src/runtime-deps) installs these same packages. You can look at those dockerfiles.
+- [pkgs.org](https://pkgs.org/) is a useful site for searching for packages, to find the one for your distribution.
+
+## Packages
+
+.NET depends on the following packages.
+
+- [GNU C Library (glibc)](https://www.gnu.org/software/libc/libc.html)
+- [GNU C++ Library](https://gcc.gnu.org/onlinedocs/libstdc++/)
+- [GCC low-level runtime library](https://gcc.gnu.org/onlinedocs/gccint/Libgcc.html)
+- [ICU](http://site.icu-project.org/)
```

Comment:
ICU is optional.

bash is (I think) not optional for dotnet.

Reply:
It isn't optional by default, but yes, I'll document invariant mode.

Is Kerberos optional?

Reply:
> Is Kerberos optional?

no